Add known issue with Kuryr Neutron ports (#1)
maxwelldb/openshift-docs: release_notes/ocp-4-6-release-notes.adoc

+
(link:https://bugzilla.redhat.com/show_bug.cgi?id=1874754[*BZ#1874754*])

> * When running on OpenStack with Kuryr, unnecessary Neutron Ports will get

Reviewer: ```suggestion
* Clusters that run on {rh-openstack} and use Kuryr create unnecessary Neutron ports
```

> created for each `hostNetworking` pod. The ports are safe to be deleted and

Reviewer: ```suggestion
for each `hostNetworking` pod. You can delete these ports safely.
```

> automatic deletion will be implemented in one of the z-stream releases of

Reviewer: ```suggestion
Automatic port deletion is planned for a future release of
```

> {product-title} 4.6.

Reviewer: ```suggestion
{product-title}.
```
